src/styles/blocks/_variables.scss

```diff
@@ -0,0 +1 @@
+$animation-time: 300ms;
```

Comment:
it's ok but remove this file from 'blocks' folder

Reply:
ok, i will move. thank you